app/tasks/doctor.php

```diff
@@ -25,36 +25,36 @@
         $domain = new Domain(App::getEnv('_APP_DOMAIN'));
 
         if(!$domain->isKnown() || $domain->isTest()) {
-            Console::log('🔴 Hostname has a public suffix');
+            Console::log('🔴 Hostname has a public suffix ('.$domain->get().')');
         }
         else {
-            Console::log('🟢 Hostname has a public suffix');
+            Console::log('🟢 Hostname has a public suffix ('.$domain->get().')');
         }
         
         $domain = new Domain(App::getEnv('_APP_DOMAIN_TARGET'));
 
         if(!$domain->isKnown() || $domain->isTest()) {
-            Console::log('🔴 CNAME target has a public suffix');
+            Console::log('🔴 CNAME target has a public suffix ('.$domain->get().')');
         }
         else {
-            Console::log('🟢 CNAME target has a public suffix');
+            Console::log('🟢 CNAME target has a public suffix ('.$domain->get().')');
         }
         
-        if(App::getEnv('_APP_OPENSSL_KEY_V1', 'your-secret-key') === 'your-secret-key') {
+        if(App::getEnv('_APP_OPENSSL_KEY_V1', 'your-secret-key') === 'your-secret-key' || empty(App::getEnv('_APP_OPENSSL_KEY_V1', 'your-secret-key'))) {
             Console::log('🔴 Using a unique secret key for encryption');
         }
         else {
             Console::log('🟢 Using a unique secret key for encryption');
         }
 
-        if(App::getEnv('_APP_ENV', 'development') === 'development') {
+        if(App::getEnv('_APP_ENV', 'development') !== 'production') {
             Console::log('🔴 App enviornment is set for production');
         }
         else {
             Console::log('🟢 App enviornment is set for production');
         }
 
-        if(App::getEnv('_APP_OPTIONS_ABUSE', 'disabled') === 'disabled') {
+        if(App::getEnv('_APP_OPTIONS_ABUSE', 'disabled') === 'disabled' || empty(App::getEnv('_APP_OPTIONS_ABUSE', 'disabled'))) {
```

Comment:
Same here :)

Reply:
The Appwrite source code has an internal default value so empty string would also be considered as disabled. I'll change the condition to `'enabled' !== ...`